docs/swagger.yml

```diff
@@ -1,2627 +1,2559 @@
-swagger: "2.0"
```

Comment:
Can you fix this commit so the whitespace doesn't change on every line?  I can help you figure out how to do that if needed, but change as is doesn't give a clear picture of what changed.

Reply:
Thanks!